sqlglot/parser.py

```diff
@@ -6366,7 +6381,7 @@ def _parse_copy(self):
 
         kind = self._match(TokenType.FROM) or not self._match_text_seq("TO")
 
-        files = self._parse_csv(self._parse_conjunction)
+        files = self._parse_csv(self._parse_file_location)
```

Comment:
Let's add a T-SQL test case. I'm not sure we should keep this as a string. Parsing a table by default is probably the better way, and if we need to preserve the string for T-SQL we could override there instead.

For example, Snowflake parses staged file paths into `Table`s that wrap around `Var`s:
```python
>>> sqlglot.parse_one("select * from @foo/bar", "snowflake")
Select(
  expressions=[
    Star()],
  from=From(
    this=Table(
      this=Var(this=@foo/bar))))
```

Reply:
You mean a test case for T-SQL's `COPY`?  If yes, I had added one in the initial commit, but I can come up with new examples to enrich them

Reply:
Ah got it, I missed that